tardis/plasma/tests/test_nlte_excitation.py

```diff
@@ -116,3 +116,24 @@ def test_prepare_bound_bound_rate_matrix(
         np.array(actual_rate_matrix),
         rtol=1e-6,
     )
+
+
+def test_coll_exc_deexc_matrix():
```

Comment:
It would be better to have the test case as a parameter, see e.g. https://github.com/tardis-sn/tardis/blob/44b2f478844c7d0af193189f68d897287be33cff/tardis/montecarlo/montecarlo_numba/tests/test_opacities.py#L7

Reply:
done